pkg/ctl/utils/associate_iam_oidc_provider.go

```diff
@@ -22,6 +22,9 @@ func associateIAMOIDCProviderCmd(cmd *cmdutils.Cmd) {
 	}
 
 	cmd.FlagSetGroup.InFlagSet("General", func(fs *pflag.FlagSet) {
+
+		fs.StringSliceVar(&cfg.IAM.OIDCClientIDList, "oidc-client-id-list", []string{}, "List of Client ID for OIDC provider (audience)")
```

Comment:
Nit: what would you think of having `oidc-client-ids` instead of `oidc-client-id-list`?